sql/parser/eval.go

```diff
@@ -316,6 +319,13 @@ var binOps = map[binArgs]binOp{
 	},
 }
 
+func inDays(d time.Duration) (DDate, error) {
+	if d%(24*time.Hour) != 0 {
```

Comment:
There is no such things as time.Day.

On Mon, Oct 12, 2015 at 3:39 PM Ben Darnell <notifications@github.com>
wrote:

> In sql/parser/eval.go
> <https://github.com/cockroachdb/cockroach/pull/2811#discussion_r41792914>:
>
> > @@ -316,6 +319,13 @@ var binOps = map[binArgs]binOp{
> >  	},
> >  }
> >
> > +func inDays(d time.Duration) (DDate, error) {
> > +	if d%(24*time.Hour) != 0 {
>
> s/24*time.Hour/time.Day/g
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/2811/files#r41792914>.
>